cloud/src/meta-service/meta_service_schema.cpp

```diff
@@ -297,6 +297,15 @@ void write_schema_dict(MetaServiceCode& code, std::string& msg, const std::strin
                << ", reached the limited size threshold of SchemaDictKeyList "
                << config::schema_dict_kv_size_limit;
             msg = ss.str();
+            return;
+        }
+        // Limit the count of dict keys
+        if (dict.column_dict_size() > config::schema_dict_key_count_limit) {
```

Comment:
also print `dict.column_dict_size()`; 
code = MetaServiceCode::KV_TXN_COMMIT_ERR;  use "invalid argument" instead

and what will happen to the existing dict, which size is larger than the config, say 2048, if we change the config to a smaller value, say, 100.

Reply:
done